### PR TITLE
monitor

### PR DESCRIPTION
**Pull request by codelinz**
**What this PR does / why we need it**:

Simplify monitoring query API

**Does this PR need to be backport to the previous release branch?**:

<!--
If no, just write "NONE".

If don't know, write "UNKNOWN", and let the reviewer decide.

If yes, write the release branches name in the below format and submit the related cherry-pick PR:
- release/3.9

cc @ioito

### CODE REVIEW

File: pkg/monitor/models/unifiedmonitor.go [hunk 1]
```diff
@@ -507,3 +508,40 @@ func fillSerieTags(series *tsdb.TimeSeriesSlice) {
 		(*series)[i] = serie
 	}
 }
+
+func (self *SUnifiedMonitorManager) GetDetailsSimpleQuery(ctx context.Context, userCred mcclient.TokenCredential, query jsonutils.JSONObject) (jsonutils.JSONObject, error) {
```
Comment (ioito) [inline pkg/monitor/models/unifiedmonitor.go:512]: query 要定义成结构体在 pkg/apis/monitor/

File: cmd/climc/shell/monitor/unifiedmonitor.go [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package monitor
+
+import (
+	"fmt"
```
Comment (zexi) [inline cmd/climc/shell/monitor/unifiedmonitor.go:4]: 这里的 import 路径需要分开下，参考文档：https://www.cloudpods.org/zh/docs/development/specification/#%E5%AF%BC%E5%85%A5%E6%A0%87%E5%87%86%E5%BA%93%E7%AC%AC%E4%B8%89%E6%96%B9%E6%88%96%E5%85%B6%E5%AE%83%E5%8C%85

File: pkg/mcclient/options/monitor/simplequery.go [hunk 3]
```diff
@@ -0,0 +1,29 @@
+package monitor
+
+import (
+	"yunion.io/x/jsonutils"
```
Comment (zexi) [inline pkg/mcclient/options/monitor/simplequery.go:4]: import (
  "yunion.io/x/jsonutils"
 
  "yunion.io/x/onecloud/pkg/apis/monitor"
)

File: pkg/mcclient/options/monitor/simplequery.go [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+	"yunion.io/x/onecloud/pkg/apis/monitor"
+)
+
+type SimpleQueryTest struct {
+	Id         string            `json:"id"`
+	NameSpace  string            `json:"name_space"`
+	MetricName string            `json:"metric_name"`
+	Starttime  string            `json:"start_time"`
```
Comment (zexi) [inline pkg/mcclient/options/monitor/simplequery.go:12]: StartTime
EndTime

File: pkg/monitor/models/unifiedmonitor.go [hunk 5]
```diff
@@ -507,3 +508,59 @@ func fillSerieTags(series *tsdb.TimeSeriesSlice) {
 		(*series)[i] = serie
 	}
 }
+
+func (self *SUnifiedMonitorManager) GetDetailsSimpleQuery(ctx context.Context, userCred mcclient.TokenCredential, query *monitor.SimpleQueryTest) (jsonutils.JSONObject, error) {
+	namespace := query.NameSpace
```
Comment (zexi) [inline pkg/monitor/models/unifiedmonitor.go:513]: 这个字段是不是叫 Database 更合理？

File: pkg/monitor/models/unifiedmonitor.go [hunk 6]
```diff
@@ -507,3 +508,59 @@ func fillSerieTags(series *tsdb.TimeSeriesSlice) {
...
+	if namespace == "" {
+		return jsonutils.JSONNull, httperrors.NewInputParameterError("not support database")
+	}
+	metricName := query.MetricName
+	metric := strings.Split(metricName, ".")
```
Comment (zexi) [inline pkg/monitor/models/unifiedmonitor.go:518]: 需要做 len(metric) == 2 的判断

**Comment by zexi** [comment]
Duplicated by https://github.com/yunionio/cloudpods/pull/15046

**Pull request closed without merging**